CHANGES.md

```diff
@@ -1,1831 +1,1552 @@
-
+# Release Notes
 
 For a list of open issues and known problems, see:
 https://github.com/radical-cybertools/radical.pilot/issues/
 
-1.33.0 Release                                                        2023-04-25
```

Comment:
please, leave it as it is. It was not a valid MD (at least not for the GH parser) and it created a ridicolous amount of 2nd order headings. Now it does not.

Reply:
I am sorry if I was not clear: I like the addition of the `current` and `past` subsections, and I also appreciate the cleaner headings.  I would like to keep the indentation for the itemization as that reads nicer, and would like to keep the text-formatting of the release headers as that reads easier in text also.  That is valid MD then - see [here](https://github.com/radical-cybertools/radical.pilot/blob/test/nb_section3/CHANGES.md) where I committed the proposed change for illustration.

Reply:
Knock yourself out with all the formatting you want. I am sure that is the best way to spend our collective hours. You will have to check and do the needful for Sphinx to render the file correctly. Otherwise, we will eliminate it from the documentation.

Reply:
Renders ok now while preserving some original formatting.  Thanks for indulging me.
https://radicalpilot.readthedocs.io/en/docs-nb_section3/release_notes.html